src/index.js

```diff
@@ -22,27 +23,25 @@ export default function RollupPluginPreprocess ({
     name: 'preprocess',
 
     /**
-     * @param {string} code
      * @param {string} fileName
-     * @returns {{ code: string, map: Object }}
+     * @returns {string}
      */
-    transform (code, fileName) {
-      if (!filter(fileName)) {
-        return;
-      }
+    load (fileName) {
+      let data = fs.readFileSync(fileName);
```

Comment:
Ouch, sorry. I forgot the encoding... Thanks!
https://github.com/Katochimoto/rollup-plugin-preprocess/releases/tag/0.0.3